_components/header/header.html

```diff
@@ -32,7 +32,7 @@
     description: Display an extended header.
 changelog:
   key: component-header
-in_page_nav: false
```

Comment:
Thanks for adding the in-page nav here. I see some issues with the heading levels here, and I'm not sure this PR is the place to fix them, but I wanted to call them out. I think the "Component preview," "Component code," "Guidance," etc., headings should be nested under each heading type (e.g., nested under "Basic header"). To me, it is not logical for these to be at the same level as the heading type--they belong under the heading type.

Reply:
@sarah-sch I opened issue #2601 to update the header structure for these scenarios.